CPS-0STL/README.md

```diff
@@ -0,0 +1,141 @@
+
+Casual perusal of information about Cardano on the internet reveals a wide variety of conflicting explanations and assertions regarding settlement and finality. The terms "settlement" and "finality" are not clearly defined in the context of Cardano and statements about them range from saying that only one block/confirmation is required to consider a transaction settled to 2160 blocks or 12 hours is required for settlement. The Ouroboros security parameter *k* also seems poorly described.
+
+> [!NOTE]
+> Are there internal IO, CF, or Intersect computations or measurements that could be included in this section?
+
+Many centralized exchanges have published the number of confirmations (blocks) required for them to consider a transaction settled. Taking Kraken as an example,[^1] we see in the following table the significant delay required for transactions to be treated as final. Cardano is not among the cluster of fast-settling blockchains. (Note that the following table contains internal contradictions between the "confirmation" column and the "time" column, a situation which further highlights imprecise understanding of settlement.)
+
+| Blockchain | Confirmations Required | Approximate time (minutes) |
+| ---------- | ---------------------: | -------------------------: |
+| Algorand   |                     10 |                          1 |
+| Aptos      |                     50 |                          5 |
+| Avalance   |                     20 |                          1 |
+| Bitcoin    |                      3 |                         30 |
+| Cardano    |                     15 |                         10 |
+| Dogecoin   |                     40 |                         40 |
+| Ethereum   |                     70 |                         14 |
+| Polkadot   |                    n/a |                          5 |
+| Ripple     |                    n/a |                          0 |
+| Solana     |                    n/a |                          0 |
+| Tezos      |                      6 |                          3 |
+[^1]: Data extracted from [https://support.kraken.com/hc/en-us/articles/203325283-Cryptocurrency-deposit-processing-times](https://support.kraken.com/hc/en-us/articles/203325283-Cryptocurrency-deposit-processing-times) on 7 August 2024.
+
+> [!NOTE]
+> Add data from other exchanges to the table above, and provide citations.
```

Comment:
Not sure how interesting more data would be here. Kraken is a very large exchange and even though others might have different numbers, they would all follow similar (unprincipled) principles.